jit/v1/jit_v1.2.0-Beta-2.yaml

```diff
@@ -0,0 +1,1312 @@
+            - $ref: '#/components/schemas/AddressLocation'
+            - $ref: '#/components/schemas/FacilityLocation'
+            - $ref: '#/components/schemas/GeoLocation'
+        eventCreatedDateTime:
+          type: string
+          x-stoplight:
+            id: bmxbd35dh3qmb
+          format: date-time
+          example: '2021-01-09T14:12:56+01:00'
+          description: "The timestamp of when the event leading to this request was created (this would be a time in the past).\r\n\r\nIf omitted the time of when the request will be recieved will be used."
+        milesToDestinationPort:
+          type: number
+          x-stoplight:
+            id: mww31on0y7350
+          format: float
+          example: 0
+          description: Remaining distance reported by the vessel to the next destination port in nautical miles.
+        eventClassifierCode:
+          type: string
+          x-stoplight:
+            id: 4barus9qz6tci
+          example: ACT
+          description: "Code for the event classifier.\r\n\r\n- PLN (Planned)\r\n- ACT (Actual)\r\n- REQ (Requested)\r\n- EST (Estimated)\r\n\r\nMore details can be found on [GitHub](https://github.com/dcsaorg/DCSA-Information-Model/blob/master/datamodel/referencedata.d/eventclassifiercodes.csv).\r\n\r\n**NB**: This is a conditional field and must match timestamps defined in the IFS."
+        operationsEventTypeCode:
+          type: string
+          x-stoplight:
+            id: r96k6isbiglrf
+          example: STRT
+          description: "The code to identify the type of event that is related to the operation\r\n\r\n- STRT (Started)\r\n- CMPL (Completed)\r\n- ARRI (Arrived)\r\n- DEPA (Departed)\r\n- OMIT (Omitted)\r\n- CANC (Cancelled)\r\n\r\nMore details can be found on [GitHub](https://github.com/dcsaorg/DCSA-Information-Model/blob/master/datamodel/referencedata.d/operationseventtypecodes.csv).\r\n\r\n**NB**: This is a conditional field and has to be aligned with other fields such as `portCallServiceTypeCode`. One such example is if `portCallServiceTypeCode` is not null then `operationsEventTypeCode` should be either: STRT (Started), COMP (Completed) or CANC (Cancelled). If `portCallServiceTypeCode` is null then `operationsEventTypeCode` is one of: ARRI (Arrived), DEPA (Departed) or OMIT (Omitted). Please check the IFS for allowed combinations."
+        modeOfTransport:
+          type: string
+          x-stoplight:
+            id: jqxvxxk1pjh9z
+          deprecated: true
+          example: VESSEL
+          description: "**NB**: this field is marked deprecated since the standard is only defined for `VESSEL` and therefore redundant.\r\n\r\nJIT 1.0 and JIT 1.1 timestamps have used this field to differentiate timestamps. Since JIT 1.2 this is no longer needed except for 1 pair of JIT 1.1 timestamps that JIT 1.2 still needs for the sake of backward compatability. These timestamps have the portCallServiceTypeCode set to SAFE."
```

Comment:
```suggestion
        modeOfTransport:
          type: string
          x-stoplight:
            id: jqxvxxk1pjh9z
          deprecated: true
          example: VESSEL
          description: "**NB**: this field is marked deprecated since the standard is only defined for `VESSEL` and therefore redundant.\r\n\r\nJIT 1.0 and JIT 1.1 timestamps have used this field to differentiate timestamps. Since JIT 1.2 this is no longer needed except for 1 pair of JIT 1.1 timestamps that JIT 1.2 still needs for the sake of backward compatability. These timestamps have the portCallServiceTypeCode set to `SAFE`."
```